modules/core/keeper/msg_server_test.go

```diff
@@ -774,3 +777,143 @@ func (suite *KeeperTestSuite) TestUpgradeClient() {
 		}
 	}
 }
+
+func (suite *KeeperTestSuite) TestChannelUpgradeTry() {
+	var (
+		path *ibctesting.Path
+		msg  *channeltypes.MsgChannelUpgradeTry
+	)
+
+	cases := []struct {
+		name      string
+		malleate  func()
+		expResult func(res *channeltypes.MsgChannelUpgradeTryResponse, err error)
```

Comment:
[nit] usually I would imagine `expResult` is the expected result, but this is an assertion function. Maybe `assertionFn`?

Reply:
that's a fair point, I think we have this test setup in some extra places in the codebase using `expResult`, so maybe we can do one swoop to change all if people feel its worth it. 

(I know we have inconsistencies with `expPass`, `expError` and `expResult` in different places - maybe we can keep some housekeeping work for a rainy day (?))

Reply:
`assertionFn` sounds nice to me. Agree it should be done in a single swoop